tests/test_completion_refresher.py

```diff
@@ -1,87 +1,82 @@
 import time
-import pytest
+import unittest
```

Comment:
Is unittest a different unit test framework? If so, why not use pytest?

Reply:
Some of our unit tests are already using unittest lib, and some are using pytest lib. They are different framework that are both supported in python. I could not able to make auto test discovery when using pytest, and it kept popping an error message saying pytest lib is not installed even though it was installed obviously. I found GitHub thread for this bug, and it seems not fixed yet. Class type unit test that extends `unittest.TestCase` (such as `test_mssqlcliclient.py`) with unittest lib are a lot more convenient to run and manage. So I switched to unittest lib as `test_mssqlcliclient.py` does.

Reply:
Sounds good. Please make sure the tests still run using the pytest command line.